api/src/main/java/org/geysermc/geyser/api/entity/EntityData.java

```diff
@@ -81,4 +81,9 @@ public interface EntityData {
      * @return whether the movement is locked
      */
     boolean isMovementLocked();
+
+    /**
+     * Allows switching the current items in the main and offhand.
```

Comment:
Let's elaborate saying this requests to the server to switch hands - it doesn't guarantee it will happen.

Reply:
i would def mention that there is no guarantee of the server accepting it